src/pudl/output/epacems.py

```diff
@@ -157,7 +158,7 @@ def epacems(
                 f"These input states are not in our dataset: {nonexistent}")
         states = list(states)
 
-    all_years = pudl.constants.WORKING_PARTITIONS['epacems']['years']
+    all_years = EpaCemsSettings['years']
```

Comment:
Why is this looking at `EpaCemsSettings['years']` rather than `EpaCemsSettings.working_partitions['years']`?

Reply:
This was just a typo, fixed now